src/raster/rasterband.jl

```diff
@@ -283,15 +283,15 @@ function copywholeraster!(
     source::T,
     dest::AbstractRasterBand;
     options = StringList(C_NULL),
-    progressdata = C_NULL,
-    progressfunc::Function = GDAL.gdaldummyprogress,
+    progressfunc::Function = progressfunc_wrapper,
+    progressdata = dummyprogress,
```

Comment:
Can we consider replacing 
```julia
    progressfunc::Function = progressfunc_wrapper,
    progressdata = dummyprogress,

[...]

        @cplprogress(_progresscallback),
        Ref(progressfunc),
```

with

```julia
    progressfunc::Function = dummyprogress,

[...]

        @cplprogress(_progresscallback),
        Ref(progressfunc),
```

everywhere?

Reply:
Yes, and removing progressdata, fully agree. But it is a breaking API change. Unless we say this was never part of the public API or this never worked anyway.

Reply:
I'm fine with it being a breaking API change

Reply:
Resolved by https://github.com/yeesian/ArchGDAL.jl/pull/353/commits/24c1304555c48b8e4931b9755e3113f35e312a52